classes/Search.php

```diff
@@ -220,5 +225,13 @@ private function set_page_title($data) {
         $DATA->set_page_metadata($this_page, 'title', $pagetitle);
     }
 
+    private function prettifySearchString($string) {
+        if ( preg_match('/speaker:(\d+)/', $string, $m) ) {
+            $MEMBER = new MEMBER(array('person_id'=>$m[1]));
+            $name = 'speaker:' . $MEMBER->full_name();
+            $string= str_replace("speaker:$m[1]", $name, $string);
```

Comment:
I thought elsewhere we had done a loop for if there were multiple speaker entries in the search string, but can't find it now, probably imagining it. Anyway, it'd be good if this replaced all speaker: references, not just one of them :)